tests/lib/store.sh

```diff
@@ -4,8 +4,12 @@ STORE_CONFIG=/etc/systemd/system/snapd.service.d/store.conf
 . $TESTSLIB/systemd.sh
 
 _configure_store_backends(){
-    systemctl stop snapd.service snapd.socket
-    mkdir -p $(dirname $STORE_CONFIG)
+    for snap in snapd.service snapd.socket; do
+        if systemctl status "$snap"; then
```

Comment:
Just call `systemctl stop snapd.service snapd.socket snapd.refresh.timer`. If any of them are running it'll stop them.